app/assets/stylesheets/markdown-editor.scss

```diff
@@ -0,0 +1,142 @@
+.md-footer,
+.md-header {
+  background: $sidebars-background;
+  border: 0;
+  display: block;
+  height: 42px;
+  margin: 0;
+  padding: 6px 6px 0;
+
+  [class^="entypo-"],
+  [class*="entypo-"],
+  .glyphicon {
+    color: $black;
+  }
+}
+
+.md-header,
+.nav-tabs {
+  border-bottom: 0;
+  margin: 0;
+  z-index: 1;
+}
+
+.md-header.btn-toolbar {
+  background-color: $background-grey;
+  overflow: hidden;
+
+  .btn-group { margin-bottom: 8px; }
+
+  @media(max-width: $screen-xs) {
+    [data-handler="bootstrap-markdown-cmdList"],
+    [data-handler="bootstrap-markdown-cmdListO"] {
+      display: none;
+    }
+
+    [data-handler="bootstrap-markdown-cmdCode"] {
+      // !important is needed to override BS' specific rules
+      // scss-lint:disable ImportantRule
+      border-bottom-left-radius: $border-radius-small !important;
+      border-top-left-radius: $border-radius-small !important;
+      // scss-lint:enable ImportantRule
+    }
+  }
+
+  [class^="entypo-"],
+  [class*="entypo-"] {
+    font-size: 13px;
+  }
+
+  [data-handler="bootstrap-markdown-cmdUrl"],
+  [data-handler="bootstrap-markdown-cmdImage"],
+  [data-handler="bootstrap-markdown-cmdList"],
+  [data-handler="bootstrap-markdown-cmdListO"],
+  [data-handler="bootstrap-markdown-cmdCode"],
+  [data-handler="bootstrap-markdown-cmdQuote"] {
+    height: 28.5px;
+    line-height: 1.25;
+    width: 33px;
```

Comment:
Please don't set the width. :(
![buttons width](https://cloud.githubusercontent.com/assets/3798614/17607917/3e06f3ba-6029-11e6-84ed-8f12ae9ba6da.png)
vs.
![buttons no width](https://cloud.githubusercontent.com/assets/3798614/17607942/5d62218a-6029-11e6-89b4-ebb6454f7e7c.png)